modules/standard/Path.chpl

```diff
@@ -457,4 +457,78 @@ proc file.realPath(): string throws {
      return ("", name);
    }
  }
+
+/* Returns the argument.  If the argument begins with `~` or `~<user>`, replace
+   that substring with that user's home directory, otherwise returns the
+   argument unchanged.
+
+   :arg path: a file or directory, which may or may not include `~` or `~<user>`
+   :type path: `string`
+
+   :return: the argument, replacing any `~` or `~<user>` with the user's home
+                directory
+   :rtype: `string`
+*/
+proc expandUser(path: string) : string
+  {
+    var i: int = 1;
+    var j: int = 1;
+    var k: int = 1;
```

Comment:
Note that Chapel allows you to declare all of these on the same line, and would allow you to infer the type from the value provided (since the value `1` defaults to having a type of `int`).  To take advantage of either of those features, you would write these variable declarations like this:
```chapel
var i, j, k = 1;
```
or
```chapel
var i, j, k: int = 1;
```